sdk/elixir/test/dagger/client_test.exs

```diff
@@ -16,7 +16,7 @@ defmodule Dagger.ClientTest do
     Sync
   }
 
-  setup do
+  setup_all do
```

Comment:
I found the slow a little bit if we run with auto-provision. So I move to setup_all to initialize only once. Since tests are for make sure our query build and graphql client work correctly, I think it should be fine, except the test that use docker build that sometimes it error due to dockerfile not found.

Reply:
Yup, sounds reasonable to me :tada: As long as you're comfortable with it, I'm comfortable with it.